html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeleton</title>
+    <body>
+      Hello, HTML!
+    </body>
```

Comment:
The `body` element should be a sibling to the `head` element, not a child of it.